static/app/components/feedback/list/mailboxPicker.tsx

```diff
@@ -21,6 +24,9 @@ export default function MailboxPicker({onChange, value}: Props) {
       >
         <SegmentedControl.Item key="unresolved">{t('Inbox')}</SegmentedControl.Item>
         <SegmentedControl.Item key="resolved">{t('Resolved')}</SegmentedControl.Item>
+        <SegmentedControl.Item key="ignored" disabled={!hasSpamFeature}>
+          {t('Spam')}
+        </SegmentedControl.Item>
```

Comment:
I think we should hide this if the feature is not available. disabling it will make it visible, but not clickable.

Reply:
something like `{hasSpamFeature ? <SegmentedControl.Item key="ignored"> : null` isn't allowed :(

Reply:
can add a tooltip like @billyvg suggested though

Reply:
Actually, since this is a ff feature pre-release, I think it'd be a bit awkward to have this disabled (even if there is a tooltip), as the user can't even do anything with that information.

I think there's a few ways you could accomplish hiding it: you could ternary by control and have 2 separate and partially duplicated lists, or have the children as a filtered array

Reply:
ah thank you so much @billyvg !!